content/blog/globally-distributed-serverless-application-in-100-lines-of-code-infrastructure-included/index.md

```diff
@@ -81,7 +97,9 @@ const fn = new azure.appservice.HttpEventSubscription("GetUrl", {
         const endpoint = cosmosdb.endpoint.get();
         const masterKey = cosmosdb.primaryMasterKey.get();
 
-        const container = await getContainer(endpoint, masterKey);
+        const client = new CosmosClient(
```

Comment:
Do you think it'd be worth adding the import for `CosmosClient` to the code block above that has the other imports?

```typescript
import { CosmosClient } from "@azure/cosmos";
```

Reply:
Great catch! I changed the wildcard import to client import (I believe it's the only usage now)